dashboard/test/models/reviewable_project_test.rb

```diff
@@ -0,0 +1,5 @@
+require 'test_helper'
+
+class ReviewableProjectTest < ActiveSupport::TestCase
```

Comment:
I don't think we typically include this in migration PRs, but can't really hurt I guess.